scripts/provider-release-tests.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Trigger the oqs-provider release tests in CI.
```

Comment:
Maybe better call this "provider-test-trigger.sh"? It's not only working for releases... Also add SPDX identifier?

Reply:
Done and done!